src/poprox_storage/repositories/articles.py

```diff
@@ -201,6 +206,24 @@ def fetch_article_by_url(self, article_url: str, newsletter_id: UUID | None = No
         else:
             return None
 
+    def fetch_entity_by_name(self, entity_name: str) -> Entity | None:
+        entity_table = self.tables["entities"]
+
+        entity_query = select(entity_table).where(entity_table.c.name == entity_name)
```

Comment:
Do we currently expect to have/need entity name string normalization?

My gut says "no" -- but I've seen this sort of code be a constant pain point in tag-data (which definitely needs normalization) so I wanted to get a second check.

Reply:
I think we're mostly okay because we're using data that's essentially exported from AP's knowledge graph. (Not saying it won't be an issue, but I'm comfortable ignoring it for now and waiting for it to bite us before trying to address it.)